proposals/readonly-ref.md

```diff
@@ -460,6 +460,7 @@ The main competing design is really "do nothing".
 * Conversions at 'in' call sites.
 * Need to agree on metadata representation.
 * Changes to the Verification rules
+* Spec 7.5.3.1 would have to change to reflect that ref extension methods arguments are now passed without using a 'ref' keyword.
```

Comment:
@svick this is the spec (5.0): https://www.microsoft.com/en-us/download/details.aspx?id=7029
This note is added so that we make sure the spec (this repository) is updated appropriately when the feature is complete.